### PR TITLE
Batch normalization sampling

### PR DESCRIPTION
**Pull request by AntoineTheb**

### CODE REVIEW

File: deep-learning/_posts/2018-10-05-batch-normalization-sampling.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    venue:   "ICLR 2019 (under review)"
+pdf: "https://openreview.net/forum?id=HkGsHj05tQ"
+---
+
+## The idea
+Batch normalization works really well but is really slow even with parralel optimizations.
+
+![](/deep-learning/images/batch_normalization_sampling/slow.png)
+
+But the authors suggest that you can speedup the process but sampling only part of the batch and estimating mean and variance. Models C, D are Resnet-18 and Model E is Resnet-50
```
Comment (Dref360) [inline deep-learning/_posts/2018-10-05-batch-normalization-sampling.md]: Buy the authors --> The authors
the process by*

File: deep-learning/_posts/2018-10-05-batch-normalization-sampling.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## The method
+
+![](/deep-learning/images/batch_normalization_sampling/sampling.png)
+
+Multiple samping methods are proposed, but the authors mostly used BS and FS. They also used Virtual Dataset Normalization where estimation is both done on virtual data and sampled data according to a predermined ratio.
```
Comment (Dref360) [inline deep-learning/_posts/2018-10-05-batch-normalization-sampling.md]: sampling*

Comment (Dref360) [inline deep-learning/_posts/2018-10-05-batch-normalization-sampling.md]: predetermined*

Comment (Dref360) [inline deep-learning/_posts/2018-10-05-batch-normalization-sampling.md]: What's FS and BS?

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,4 @@ _previews
 .sass-cache
 .jekyll-metadata
 .Rhistory
+*.swp
```
Comment (ppoulin91) [inline .gitignore]: Why are there swap files in your repo??

File: deep-learning/_posts/2018-10-05-batch-normalization-sampling.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+layout: review
+title: "Batch normalization sampling"
+tags: deep-learning
```
Comment (ppoulin91) [inline deep-learning/_posts/2018-10-05-batch-normalization-sampling.md]: Could use other tags : optimization, layers

File: .gitignore (hunk 3 above)
Comment (AntoineTheb) [reply .gitignore]: Vim temporary files

**Comment by AntoineTheb** [comment]
all fixed @Dref360
